scripts/cargo-rustodon

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -euo pipefail
+
+exec scripts/run
```

Comment:
This shim script is here mainly to allow `cargo watch` to continue to do the right thing while allowing us to infer the right build features and set them at runtime.